praw/models/reddit/modmail.py

```diff
@@ -0,0 +1,52 @@
+from ...const import API_PATH, urlparse
+from ...exceptions import ClientException
+from .base import RedditBase
+
+
+class Modmail(object):
+    """Provides modmail functions for a subreddit."""
+    def __init__(self, subreddit):
+        self.subreddit = subreddit
+
+    def conversation(self,  # pylint: disable=invalid-name,redefined-builtin
```

Comment:
At first glance maybe this would be good as a `__call__` function so that one needn't do `modmail.conversation(...)` but only `modmail(...)`. Thoughts?